src/DefaultTheme.js

```diff
@@ -7,135 +7,716 @@
+import moduleClasses             from './Module/module.css';
+import navBarClasses             from './NavBar/navBar.css';
+import navDropdownClasses        from './NavDropdown/navDropdown.css';
+import navItemClasses            from './NavItem/navItem.css';
+import navListClasses            from './NavList/navList.css';
+import nessieLogoClasses         from './NessieLogo/nessieLogo.css';
+import notificationBarClasses    from './NotificationBar/notificationBar.css';
+import pageClasses               from './Page/page.css';
+import pageContentClasses        from './PageContent/pageContent.css';
+import pageContentHeaderClasses  from './PageContentHeader/pageContentHeader.css';
+import pageFooterClasses         from './PageFooter/pageFooter.css';
+import pageHeaderClasses         from './PageHeader/pageHeader.css';
+import paginatorClasses          from './Paginator/paginator.css';
+import progressBarClasses        from './ProgressBar/progressBar.css';
+import progressIndicatorClasses  from './ProgressIndicator/progressIndicator.css';
+import radioClasses              from './Radio/radio.css';
+import scrollBarClasses          from './ScrollBar/scrollBar.css';
+import scrollBoxClasses          from './ScrollBox/scrollBox.css';
+import sectionClasses            from './Section/section.css';
+import sliderClasses             from './Slider/slider.css';
+import sliderGroupClasses        from './SliderGroup/sliderGroup.css';
+import sorterClasses             from './Sorter/sorter.css';
+import spinnerClasses            from './Spinner/spinner.css';
+import statusIndicatorClasses    from './StatusIndicator/statusIndicator.css';
+import switchClasses             from './Switch/switch.css';
+import tabClasses                from './Tab/tab.css';
+import tabButtonClasses          from './TabButton/tabButton.css';
+import tableClasses              from './Table/table.css';
+import tableCellClasses          from './TableCell/tableCell.css';
+import tableRowClasses           from './TableRow/tableRow.css';
+import tabsClasses               from './Tabs/tabs.css';
+import tagClasses                from './Tag/tag.css';
+import tagInputClasses           from './TagInput/tagInput.css';
+import textClasses               from './Text/text.css';
+import textAreaClasses           from './TextArea/textArea.css';
+import textInputWithIconClasses  from './TextInputWithIcon/textInputWithIcon.css';
+import timeInputClasses          from './DatePicker/timeInput.css';
+import toggleButtonClasses       from './ToggleButton/toggleButton.css';
+import tooltipClasses            from './Tooltip/tooltip.css';
+import uploaderClasses           from './Uploader/uploader.css';
+import valuedTextInputClasses    from './ValuedTextInput/valuedTextInput.css';
+import withDropdownClasses       from './Addons/withDropdown/withDropdown.css';
+import withStickyClasses         from './Addons/withSticky/withSticky.css';
 
```

Comment:
Great work! 💣 
However, there are couple of components that appear to be broken after this; it could be connected with either changes that are here or it may require some changes in `defaults.json` (I'm not sure if these were broken before or after these changes 🤔 ):
- CheckableGroup
- CheckboxGroup
- DatePicker (TimeInput)
- DimensionsInput (size when focused)
- Fieldset
- IconButton
- ListBox
- NessieLogo (nothing is displayed 🤔)
- Page
- RadioGroup
- Slider (some stepLabels look like they're cut)
- SliderGroup
- Tab
- TabButton
- Tabs (margins)
- Tag (margins)
- TagInput (Tag?)
- TextArea (margins/paddings)
- ValuedTextInput (minor style issues)

Could be that only few of these need changes and others will then update to correct styles

Reply:
Thanks for the thorough checks! 😅 

> - CheckableGroup
> - CheckboxGroup
> ...
> - RadioGroup
> ...
> - Fieldset

Fixed [here](https://github.com/sociomantic-tsunami/nessie-ui/pull/735/commits/178d1b848974a648867634e4e55ee7b352efed13)!

> - DatePicker (TimeInput)

Fixed [here](https://github.com/sociomantic-tsunami/nessie-ui/pull/735/commits/2300bfb0a8f89defdc3aa94d678ce7624727f6bc)!

> - DimensionsInput (size when focused)

This didn’t change (seems this is a[ feature not a bug](https://github.com/sociomantic-tsunami/nessie-ui/issues/697) 😑) 

> - IconButton

Seems fine to me 🤔 Can you let me know the exact problem?

> - ListBox

Fixed [here](https://github.com/sociomantic-tsunami/nessie-ui/pull/735/commits/1e4ab452d43c8aefe9b1bb5520271eb95caa1cc8)!

> - NessieLogo (nothing is displayed 🤔)

Ah this has been broken for a long time `[res-wontfix]` 😄

> - Page

No change

> - Tab
> - TabButton
> - Tabs (margins)

Seems [these changes](https://github.com/sociomantic-tsunami/nessie-ui/pull/633/files) got lost 😕 Fixed [here](https://github.com/sociomantic-tsunami/nessie-ui/pull/735/commits/e42641b951ea00890d790c47de0e95f897e9961b)!

> - TagInput (Tag?)

Fixed [here](https://github.com/sociomantic-tsunami/nessie-ui/pull/735/commits/7475433b1ae33efa6e570af73bff0667062d2cb3)!

> - TextArea (margins/paddings)

Fixed [here](https://github.com/sociomantic-tsunami/nessie-ui/pull/735/commits/55164f3da3b4e10938c6a89ac5acc16fd71499ac)! (the component was completely unstyled 😱)

> - ValuedTextInput (minor style issues)

Fixed [here](https://github.com/sociomantic-tsunami/nessie-ui/pull/735/commits/fb8b4f48e40aaaba560a4bb9b050c7541a60be0d)!

Reply:
Nice! ✨
One minor thing left(-over; from [here](https://github.com/sociomantic-tsunami/nessie-ui/commit/addcdaa8e5b99e08dc24fe0c93a9460e2b787fc2#diff-88f71abda393a5147a5aeb95134f7bb5) ) - on `line 148` in `SliderGroup/index.js`, the comment is outside of closing tag and it breaks the rest of code; can you please fix this too? 😁 

Other than that, ready to be approved 👍